### ISSUE TITLE
some arguments passed to convert need to precede the source filename

### ISSUE THREAD

**Issue by jsierles** [markevans/dragonfly#391]
Some arguments, such as 'density' for specifying a source PDF file's resolution, must come before the source filename, otherwise they are ignored. This is important for controlling the target resolution of a vector image converted to PNG.

BAD
% convert test.pdf -density 150 test.png

GOOD
% convert -density test.pdf test.png

Should a PR for this should include an extra option for these arguments?

**Comment by markevans** [comment]
Yes please. I won't have a chance to look at it for a few weeks but will do
then

**Comment by trevvvy** [comment]
any progress on this PR? I'd also appreciate this option.
